test/v3/createPool/weighted/weighted.validation.test.ts

```diff
@@ -24,12 +24,14 @@ describe('create weighted pool input validations', () => {
                     weight: parseEther(`${1 / 2}`),
                     rateProvider: zeroAddress,
                     tokenType: TokenType.STANDARD,
+                    paysYieldFees: false,
```

Comment:
reason here being improved redeability?

Reply:
reason here is that I changed `paysYieldFees` from optional to required for all v3 pool creation inputs